modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,88 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-kuryr"]
+:osp-kuryr:
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user-kuryr"]
+:osp-kuryr:
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-restricted"]
+:osp-ipi:
+:osp-restricted:
+endif::[]
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
+
+You can install {product-title} on {rh-openstack-first} without using a floating IP address and external network. In the
```

Comment:
if no external_network, then the bootstrap server will not got the ignition file from OSP, detail refer to https://bugzilla.redhat.com/show_bug.cgi?id=1878758#c11